modules/openy_features/openy_webform_demo/config/install/webform.webform.demo_form.yml

```diff
@@ -0,0 +1,102 @@
+langcode: en
```

Comment:
This form should be treated as demo content. It should live in appropriate submodule(https://github.com/ymcatwincities/openy/tree/8.x-1.x/modules/openy_features/openy_demo_content/modules) and should be enabled during installation.